### DISCUSSION TITLE
Why is opening files so much slower than in ranger?

### DISCUSSION THREAD

**Discussion by kirusfg** [gokcehan/lf#813] [Q&A] [answered]
As per title, I have noticed a significant delay between opening a pdf file and zathura launching when I switched from ranger over to lf. This delay became less painful once I replaced my slow HDD with an SSD, but lf is still noticeably slower than ranger in this regard.

Are there ways to mitigate this slight annoyance? If not, at least knowing the reason will satisfy me.

**Comment by devnull09** [comment, marked as answer]
By default, lf uses xdg-open to start external applications.
I noticed a slight delay as well and solved it by setting $OPENER to a different program.

Instead of xdg-open, you can use "handlr" or rangers "rifle"

Just include this in your .bashrc (or whichever shell config you use)
`export OPENER=rifle`
